docs/concepts/what-is-an-activity-execution.md

```diff
@@ -14,20 +14,37 @@ An Activity Execution is the full chain of [Activity Task Executions](/concepts/
 
 ![Activity Execution](/diagrams/activity-execution.svg)
 
-An Activity Execution has no time limit.
-Activity Execution time limits and retries can be optimized for each situation within the Temporal Application.
+By default, an Activity Execution has no time limit.
+Activity Execution [timeouts](/application-development/features#activity-timeouts) and [retry policies](/concepts/what-is-a-retry-policy) can be customized.
 
-If for any reason an Activity Execution does not complete (exhausts all retries), the error is returned to the [Workflow](/workflows), which decides how to handle it.
+If an Activity Execution fails (because it exhausted all retries, threw a [non-retryable error](/concepts/what-is-a-retry-policy#non-retryable-errors), or was canceled), the error is returned to the [Workflow](/workflows), which decides how to handle it.
 
-### Request Cancellation
+### Cancellation
 
-A Workflow can request to cancel an Activity Execution.
-When an Activity Execution is canceled, or its Workflow Execution has completed or failed, the context passed into its function is canceled, which also sets its channel’s closed state to `Done`.
-An Activity can use that to perform any necessary cleanup and abort its execution.
+Activity Cancellation:
 
-Cancellation requests are only delivered to Activity Executions that Heartbeat:
+- lets the Activity know it doesn't need to keep doing work, and
+- gives the Activity time to clean up any resources it has created.
 
-- The Heartbeat request fails with a special error indicating that the Activity Execution is canceled.
-  Heartbeats can also fail when the Workflow Execution that spawned it is in a completed state.
-- The Activity should perform all necessary cleanup and report when it is done.
-- The Workflow can decide if it wants to wait for the Activity cancellation confirmation or proceed without waiting.
+Activities can only receive Cancellation if they emit Heartbeats or are Local Activities (which can't heartbeat but receive Cancellation anyway).
```

Comment:
```suggestion
Activities can only receive Cancellation if they emit Heartbeats or are Local Activities (which don't heartbeat but receive Cancellation anyway).
```

Also, this isn't _strictly_ true as there's an implied heartbeat, but I think the phrasing here is just fine.